conda_smithy/configure_feedstock.py

```
@@ -384,7 +384,7 @@ def main(forge_file_directory):
     build_config.CONDA_PY = 10
 
     recipe_dir = 'recipe'
-    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
+    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash -ex'},
```

Yeah, I had a lot of silent failure without these settings